app/routes.js

```diff
@@ -1,13 +1,18 @@
 // @flow
 import React from 'react';
-import { Route } from 'react-router';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Switch, Route } from 'react-router';
 import App from './containers/App';
 import HomePage from './containers/HomePage';
 import CounterPage from './containers/CounterPage';
 
-export default (
-  <Route path="/" component={App}>
-    <Route path="/" component={HomePage} />
-    <Route path="/counter" component={CounterPage} />
-  </Route>
+export default () => (
+  <Router>
+    <App>
```

Comment:
I was pretty sure that `<Router component={App}>` should now be `<Router><App>` as it is always within App, but both styles seem to work. Do prefer `<Router component={App}>`  ?